src/client_side.cc

```diff
@@ -3993,8 +3993,16 @@ ConnStateData::unpinConnection(const bool andClose)
 }
 
 void
-ConnStateData::terminateAll(const Error &error, const LogTagsErrors &lte)
+ConnStateData::terminateAll(const Error &rawError, const LogTagsErrors &lte)
 {
+    auto error = rawError; // (cheap) copy so that we can detail
+    // We detail even ERR_NONE: There should be no transactions left, and
+    // detailed ERR_NONE will be unused. Otherwise, this detail helps in triage.
+    if (!error.detail) {
+        static const auto d = MakeNamedErrorDetail("WITH_CLT");
```

Comment:
* `CLT` is what the existing official code is using to mark client-Squid processing side when detailing errors: CLT_REQMOD_ABORT, CLT_REQMOD_RESP_BODY, CLT_REQMOD_REQ_BODY. This is why this PR is using that abbreviation to detail error processing context.
* The existing official code is using `CLIENT` when referring to the protocol agent (rather than indicating processing side):  ERR_CLIENT_GONE and TCP_CLIENT_REFRESH_MISS.

@yadij, should we keep using that abbreviation for consistency with existing processing details or switch to `CLIENT` and perhaps rename the existing details that are using `CLT` (in a separate PR)?